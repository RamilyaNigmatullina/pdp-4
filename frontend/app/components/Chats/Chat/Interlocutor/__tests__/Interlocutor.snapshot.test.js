import React from 'react';
import { shallow } from 'enzyme';
import Interlocator from '../index';

describe('Interlocutor', () => {
  let props;

  const render = () => shallow(<Interlocator {...props} />);

  beforeEach(() => {
    props = {
      interlocutor: {
        id: 1,
        full_name: 'John Smith',
        avatar: '/john_smith_avatar.jpg',
      },
    };
  });

  it('renders interlocutor info', () => {
    const component = render();

    expect(component).toMatchSnapshot();
  });
});
