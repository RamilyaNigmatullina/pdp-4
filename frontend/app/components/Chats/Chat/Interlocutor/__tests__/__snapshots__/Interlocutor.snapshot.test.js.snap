// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Interlocutor renders interlocutor info 1`] = `
<div
  className="interlocutor"
>
  <div
    className="interlocutorAvatar"
  >
    <img
      alt=""
      className="rounded-circle"
      height="32"
      src="/john_smith_avatar.jpg"
      width="32"
    />
  </div>
  <div
    className="interlocutorFullName"
  >
    John Smith
  </div>
</div>
`;
